core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -378,6 +378,7 @@ case class FormattedQualificationSummaryInfo(
     nestedComplexTypes: String,
     potentialProblems: String,
     longestSqlDuration: Long,
+    totalStageWallClockDuration: Long,
```

Comment:
the name is a little bit confusing.
Once you sum "durations" together, the result won't be a wallclock any longer.